File: Projekti/JDBC-Benchmark/src/main/java/com/example/JDBCBenchmark/util/ResultWriter.java
package com.example.JDBCBenchmark.util;

public class ResultWriter {

    public static void writeHeader(String query){
        System.out.println("=================================================================");
        System.out.println(" Query: " + query);
        System.out.println(" ----------------------------------------------------------------");
    }

    public static void writeFooter(int count, long time){
        System.out.println(" Total number of records: " + count);
        System.out.println(" Elapsed time: " + time + " ms.");
        System.out.println("=================================================================");
    }

    public static void writeAverageFooter(double average) {
        System.out.println("Finished running tests for query.");
        System.out.println(" Elapsed time: " + Math.round(average*100)/100 + " ms.");
    }
}

File: Projekti/EntityFramework-Benchmark/Settings/ResultWriter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EntityFramework_Benchmark.Settings
{
    public class ResultWriter
    {
        public static void WriteHeader(string query)
        {
            Console.WriteLine("=================================================================");
            Console.WriteLine(" Query: " + query);
            Console.WriteLine(" ----------------------------------------------------------------");
        }

        public static void WriteFooter(int count)
        {
            Console.WriteLine(" Total number of records: " + count);

            Console.WriteLine(" Elapsed time: " + BStopwatch.ElapsedMilliseconds() + " ms.");
            Console.WriteLine("=================================================================");
        }

        public static void WriteElapsedTime(int i, long elapsedTime)
        {
            Console.WriteLine(i + ". Elapsed time for query: " + elapsedTime);
        }

        public static void WriteAverageFooter(double average)
        {
            Console.WriteLine(" Average elapsed time: " + Math.Round(average, 2) + "ms.");
            Console.WriteLine("=================================================================");
        }
    }
}
